docker/local-universe/Makefile

```diff
@@ -1,4 +1,7 @@
 REPO_BASE_DIR ?= $(shell pwd -P)/../..
+static_version ?= 0.2
```

Comment:
This can be overridden on the command-line by doing e.g. `make static_version=0.3 <target>`